blocks/api/serializer.js

```diff
@@ -93,9 +93,11 @@ export function getCommentAttributes( allAttributes, attributesFromContent ) {
 			const contentValue = attributesFromContent[ key ];
 
 			// save only if attribute if not inferred from the content and if valued
-			return ! ( contentValue !== undefined || allValue === undefined )
-				? Object.assign( toSave, { [ key ]: allValue } )
-				: toSave;
+			if ( ! isNil( contentValue ) || isNil( allValue ) ) {
+				return toSave;
+			}
+
+			return Object.assign( toSave, { [ key ]: allValue } );
```

Comment:
You mean switching away from `! ( ... ) ? ... : ...`? I found it easier to read, sorry. :)